docker-compose.yaml

```diff
@@ -145,6 +145,23 @@ services:
       - "9090:9090"
     extra_hosts:
       - "host.docker.internal:host-gateway"
+  tempo:
+    image: grafana/tempo:latest
+    ports:
+      - "4318:4318" # OTLP HTTP ingest
```

Comment:
@Nagaprasadvr once deployed in nomad this port will vary. How is it configured on das api?